libs/entry-components/styles/modules/vendors/rich-text/necessary-mix/_buttons-generator.scss

```diff
@@ -0,0 +1,46 @@
+@use 'sass:map';
+@use '@angular/material' as mat;
+
+$default-material-button: map.get(mat.define-typography-config(), 'button');
+
+@mixin generate-from($theme) {
+	$theming-buttons: map.get($theme, 'general', 'fonts', 'buttons');
+
+	.enigmatry-redactor-content {
+		button, .rx-content button {
+			
+			@if (theming-buttons, 'colors', 'primary') {
```

Comment:
That's fine. Short and sweat, but where did you find it? Nothing about that in official docs AFAIK?

Reply:
The ability to pass multiple keys to map.get is a feature supported in Dart Sass, as described in the official Sass documentation. This functionality allows for concise access to deeply nested values in maps without chaining multiple map.get calls manually. It simplifies the process of working with complex nested data structures like our $theme map

This feature is documented in [the official Sass sass:map module](https://sass-lang.com/documentation/modules/map/)

Additionally, here are GitHub issuse tracking the feature:
https://github.com/sass/sass/issues/1739
https://github.com/sass/dart-sass/issues/1884

Reply:
This is not about map.get keys. We're using that on various places. The question was about if condition joined with such map syntax?
IDK how this would even compile: @if ($theming-buttons, 'colors', 'primary')

Is the intention to map.get primary color and then compare it with unset (null) value?